tests/package.json

```diff
@@ -19,7 +19,7 @@
     "chai": "^4.2.0",
     "mocha": "^8.1.3",
     "mocha-steps": "^1.3.0",
-    "moonbeam-types-bundle": "^1.0.5",
+    "moonbeam-types-bundle": "^1.0.6",
```

Comment:
@crystalin is there a way to set this equal to path like in rust, like "../moonbeam-types-bundle"? If not, it'll be difficult to test whether the integration tests work with the changes without making the update and running the tests.

Reply:
Yes you can use the file:../../moonbeam-types-bundle I think